src/main/java/frc/robot/RobotContainer.java

```diff
@@ -36,40 +25,51 @@ public class RobotContainer {
   private final ControllerSubsystem controls = new ControllerSubsystem();
   private final OnebarSubsystem onebar = new OnebarSubsystem();
   private final LimelightSubsystem limelightSubsystem = new LimelightSubsystem();
+  private final WristSubsystem wristSubsystem = new WristSubsystem();
+  private final ClawSubsystem clawSubsystem = new ClawSubsystem();
 
   // Defining Commands
+  // Drivetrain
   private final AutoCmd m_autoCommand = new AutoCmd();
   private final TeleopCmd teleopCmd = new TeleopCmd(drivetrain);
   private final ShiftdownCmd shiftdown = new ShiftdownCmd(drivetrain);
   private final ShiftupCmd shiftup = new ShiftupCmd(drivetrain);
-
   private final MoveToAprilTagCmd moveToAprilTagCmd =
       new MoveToAprilTagCmd(drivetrain, limelightSubsystem);
 
+  // OneBar
   private final OnebarDown armDown = new OnebarDown(onebar);
   private final OnebarUp armUp = new OnebarUp(onebar);
   private final OnebarOut armOut = new OnebarOut(onebar);
   private final OnebarIn armIn = new OnebarIn(onebar);
   private final ArmExtStop armExtStop = new ArmExtStop(onebar);
   private final ArmRotStop armRotStop = new ArmRotStop(onebar);
-  private final WristSubsystem wristSubsystem = new WristSubsystem();
+
+  // Wrist
   private final LowerCmd lowerCmd = new LowerCmd(wristSubsystem);
   private final RaiseCmd raiseCmd = new RaiseCmd(wristSubsystem);
   private final RotLeftCmd rotLeftCmd = new RotLeftCmd(wristSubsystem);
   private final RotRightCmd rotRightCmd = new RotRightCmd(wristSubsystem);
   private final StopRaiseCmd stopRaiseCmd = new StopRaiseCmd(wristSubsystem);
   private final StopRotCmd stopRotCmd = new StopRotCmd(wristSubsystem);
-  private final ClawSubsystem clawSubsystem = new ClawSubsystem();
+
+  // Claw
   private final ClampCmd clampCmd = new ClampCmd(clawSubsystem);
   private final ReleaseCmd releaseCmd = new ReleaseCmd(clawSubsystem);
   private final ShiftLeftCmd shiftLeftCmd = new ShiftLeftCmd(clawSubsystem);
   private final ShiftRightCmd shiftRightCmd = new ShiftRightCmd(clawSubsystem);
   private final StopClawCmd stopClawCmd = new StopClawCmd(clawSubsystem);
 
-  public Trigger supplier(int buttonID) {
-    BooleanSupplier bsup = () -> driver1.getRawButton(buttonID);
-    Trigger mybutton = new Trigger(bsup);
-    return mybutton;
+  public Trigger supplier(int buttonID, boolean driver) {
```

Comment:
Can you make the new variable an enum instead with options for DRIVER or OPERATOR for clarity?